examples/hbjs.example.html

```diff
@@ -3,7 +3,7 @@
 <script>
 // instantiateStreaming isn't supported in Safari yet, see nohbjs.html
 WebAssembly.instantiateStreaming(fetch("../hb.wasm")).then(function (result) {
-  result.instance.exports.memory.grow(400); // each page is 64kb in size
+  //result.instance.exports.memory.grow(400); // each page is 64kb in size
```

Comment:
I think just take these out.

Reply:
I did that initially then since I don’t know why this was needed, thought I’d keep them just in case I found out what they mean, but I guess just getting ride of them is good.

Reply:
Previous versions of the wasm runner didn't automatically grow memory, so this was needed.